packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/pbrMaterialPropertyGridComponent.tsx

```diff
@@ -353,6 +377,16 @@ export class PBRMaterialPropertyGridComponent extends React.Component<IPBRMateri
                         step={0.01}
                         onPropertyChangedObservable={this.props.onPropertyChangedObservable}
                     />
+                    <SliderLineComponent
+                        lockObject={this.props.lockObject}
+                        label="Base Diffuse Roughness"
+                        target={material}
+                        propertyName="baseDiffuseRoughness"
+                        minimum={0}
+                        maximum={1}
+                        step={0.01}
+                        onPropertyChangedObservable={this.props.onPropertyChangedObservable}
+                    />

```

Comment:
We shouldn't have this slider in two different sections "Lighting & Colors" and "Metallic Workflow", it could be confusing for the user. Maybe keep it only in the latter, as "Roughness" is to be found in that section?

Reply:
Yeah, I'm not sure how I managed that... :)